extensions/hp-extension/src/hpHeart.ts

```diff
@@ -0,0 +1,245 @@
+          seriesMatchingRules: [
+            {
+              weight: 10,
+              attribute: 'SeriesDescription',
+              constraint: {
+                contains: 'CINE',
+              },
+            },
+            {
+              weight: 10,
+              attribute: 'SeriesDescription',
+              constraint: {
+                contains: 'LAX',
+              },
+            },
+            {
+              weight: 10,
+              attribute: 'SeriesDescription',
+              constraint: {
+                contains: '4Ch',
+              },
+            },
+          ],
+          studyMatchingRules: [],
+        },
+        {
+          id: 'displaySetCineLax3',
+          // Matches displaysets, NOT series
+          seriesMatchingRules: [
+            {
+              weight: 10,
+              attribute: 'SeriesDescription',
+              constraint: {
+                contains: 'CINE',
+              },
+            },
+            {
+              weight: 10,
+              attribute: 'SeriesDescription',
+              constraint: {
+                contains: 'LAX',
+              },
+            },
+            {
+              weight: 10,
+              attribute: 'SeriesDescription',
+              constraint: {
+                contains: '3Ch',
+              },
+            },
+          ],
+          studyMatchingRules: [],
+        },
+        {
+          id: 'displaySetCineLax2',
+          // Matches displaysets, NOT series
+          seriesMatchingRules: [
+            {
+              weight: 10,
+              attribute: 'SeriesDescription',
+              constraint: {
+                contains: 'CINE',
+              },
+            },
+            {
+              weight: 10,
+              attribute: 'SeriesDescription',
+              constraint: {
+                contains: 'LAX',
+              },
+            },
+            {
+              weight: 10,
+              attribute: 'SeriesDescription',
+              constraint: {
+                contains: '2Ch',
+              },
+            },
+          ],
+          studyMatchingRules: [],
+        },
+        {
+          id: 'displaySetCineSax',
+          // Matches displaysets, NOT series
+          seriesMatchingRules: [
+            {
+              weight: 10,
+              attribute: 'SeriesDescription',
+              constraint: {
+                contains: 'CINE',
+              },
+            },
+            {
+              weight: 10,
+              attribute: 'SeriesDescription',
+              constraint: {
+                contains: 'SAX',
+              },
+            },
+          ],
+          studyMatchingRules: [],
+        },
+      ],
+      viewports: [
+        {
+          viewportOptions: {
+            toolGroupId: 'default',
+            customViewportOptions: {
+              initialScale: 2.5,
+            },
+            // syncGroups: [
+            //   {
+            //     type: 'cameraPosition',
```

Comment:
cameraPosition is NOT the right sync group as it does really weird things to the images.  Need to see if we can fix this